itest.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
+#
+#/ Usage: itest.sh [opts] [test_pattern]
+#/
+#/ Options:
+#/    --gh-ost-binary | -b path to gh-ost binary. defaults to /tmp/gh-ost-test
+#/    --test-dir           path to test directories. defaults to /localtests
+
+usage() {
+  code="$1"
+  u="$(grep "^#/" "$0" | cut -c"4-")"
+  if [ "$code" -ne 0 ]; then echo "$u" >&2; else echo "$u"; fi
+
+  exit "$code"
+}
+
+GHOST_BINARY=/tmp/gh-ost-test
+TEST_DIR=/localtests
+
+while [ "$#" -gt 0 ];
+do
+  case "$1" in
+    --gh-ost-binary|-b) GHOST_BINARY="$2"; shift 2;;
+    --gh-ost-binary=*) GHOST_BINARY="$(echo "$1" | cut -d"=" -f"2-")"; shift;;
+    --test-dir) TEST_DIR="$2"; shift 2;;
+    --test-dir=*) TEST_DIR="$(echo "$1" | cut -d"=" -f"2-")"; shift;;
+    --help|-h) usage 0;;
+    -*) usage 1;;
+    *) break ;;
+  esac
+done
+
+TEST_PATTERN="${1:-.}"
+
+if [ -f /mysql_version ];
+then
+  MYSQL_VERSION="$(cat /mysql_version)"
+else
+  MYSQL_VERSION="5.7.26"
+fi
+
+echo "Creating replication sandbox"
+dbdeployer deploy replication $MYSQL_VERSION \
```

Comment:
This sets up one master, two replicas (the default for `dbdeployer` replication setup). Let's save time and memory and set up a single replica, instead. Again, see https://github.com/github/gh-ost/blob/master/script/cibuild-gh-ost-replica-tests:

https://github.com/github/gh-ost/blob/4dab06e92b20954b46301e3b176d3a873615e3af/script/cibuild-gh-ost-replica-tests#L43